ci-runner-image/Dockerfile

```diff
@@ -36,6 +36,12 @@ RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add -
 RUN add-apt-repository --yes --update "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" \
     && apt install -y terraform
 
+
+# Requirements to manage EC2 instances using SSM. This should be moved into the fargate-runner-action Dockerfile \
```

Comment:
I'd say we can leave `# Requirements to manage EC2 instances using SSM`

Reply:
Thanks 👁️  done in https://github.com/newrelic/fargate-runner-action/pull/30/commits/492b236077542cd66e4044ebbc5f5e0d784d1a04